tests/acceptance_crontab_spec.lua

```diff
@@ -0,0 +1,216 @@
+describe("acceptance: crontab file pattern", function()
+    local cronex
+    local buf
+    local orig_buf
+    
+    -- Helper function to wait for async operations
+    local function wait(ms)
+        local co = coroutine.running()
+        vim.defer_fn(function()
+            coroutine.resume(co)
+        end, ms or 1000)
+        coroutine.yield()
+    end
+    
+    describe("with quoted cron expressions", function()
+        -- Setup before each test
+        before_each(function()
+            -- Save current buffer to restore later
+            orig_buf = vim.api.nvim_get_current_buf()
+            
+            -- Create mock buffer with .crontab extension
+            buf = vim.api.nvim_create_buf(false, true)
+            vim.api.nvim_buf_set_name(buf, "test.crontab")
+            vim.api.nvim_buf_set_lines(buf, 0, -1, false, {
+                "'* * * * *' command",
+                "# this line has no cron",
+            })
+            
+            -- Load but don't setup yet
+            cronex = require("cronex")
+        end)
+        
+        -- Cleanup after each test
+        after_each(function()
+            -- Disable plugin and clear diagnostics
+            vim.cmd("CronExplainedDisable")
+            
+            -- Reset buffer to original
+            vim.api.nvim_set_current_buf(orig_buf)
+            pcall(vim.api.nvim_buf_delete, buf, { force = true })
+        end)
+        
+        it("Activates plugin for .crontab files", function()
+            -- Setup plugin with .crontab in file_patterns
+            cronex.setup({
+                file_patterns = { "*.crontab" },
+                explainer = {
+                    cmd = { "echo", "test-explanation" }
+                }
+            })
+            
+            -- Set current buffer and manually trigger the BufEnter autocmd
+            vim.api.nvim_set_current_buf(buf)
+            vim.cmd("doautocmd BufEnter")
+            
+            -- Wait for async operations to complete
+            wait()
+            
+            -- Get plugin namespace
+            local ns = vim.api.nvim_get_namespaces()["plugin-cronex.nvim"]
+            assert.is_not_nil(ns, "Plugin namespace should exist")
+            
+            -- Get diagnostics
+            local diags = vim.diagnostic.get(buf)
```

Comment:
Can we be more specific here with 
```lua
vim.diagnostic.get(buf, { namespace = ns })
```?

So that we make sure we look at the diagnostics that we care about. Otherwise the assertion of lines 65,66 could pass with any other present diagnostics.